server/jamovi/server/dataset/duckdataset.py

```diff
@@ -529,20 +972,21 @@ def row_count(self) -> int:
 
     @property
     def row_count_ex_filtered(self) -> int:
-        # TODO
-        return self.row_count
+        return self._row_count_ex_filtered
 
     @property
     def weights(self) -> int:
-        result = self._execute(f"""
-            SELECT ivalue
-            FROM "sheet_meta_{ self._id }"
-            WHERE name = 'weights'
-            """).fetchall()
-        (w,) = next(iter(result))
-        return w
+        return self._weights
+        # result = self._execute(f"""
+        #     SELECT ivalue
+        #     FROM "sheet_meta_{ self._id }"
+        #     WHERE name = 'weights'
+        #     """).fetchall()
+        # (w,) = next(iter(result))
+        # return w
```

Comment:
Can this be removed?

Reply:
i will want to use it later ... which is why i was keeping it around